typings.json

```diff
@@ -1,6 +1,8 @@
 {
-  "ambientDependencies": {
+  "globalDependencies": {
     "es6-shim": "registry:dt/es6-shim#0.31.2+20160317120654",
-    "jasmine": "registry:dt/jasmine#2.2.0+20160412134438"
+    "core-js": "registry:dt/core-js#0.0.0+20160602141332",
```

Comment:
That is correct, I actually remove esh6-shim it will throw an error if both are included. It should be part of the commit, dont see why this was included.